bindata/network/network-metrics/003-prometheus_rules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: network-metrics-with-network-name
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  namespace: openshift-monitoring
+  labels:
+    app: network-metrics
+    prometheus: k8s
+    role: alert-rules
+spec:
+  groups:
+    - name: network_with_name
+      rules:
+        - record: network:container_network_receive_bytes_total
+          expr: (container_network_receive_bytes_total) + on(namespace,pod,interface) group_left(network_name) ( pod_network_name_info )
```

Comment:
Are you sure there are no selectors needed for these? 

Also curious what is pod_network_name_info?

Reply:
Also, is the left hand side here supposed to be the cAdvisor metric? A typical join should also be done with multiplications.

Reply:
I'm also missing why we need these recording rules, the join is pretty straight forward, for it to be done in ad-hoc querying. I'm saying that as this will add `# of containers * number of interfaces per container` number of time-series, which is pretty significant.

Reply:
Will try to reply to all in one:
- `pod_network_name_info` is a dumb metric that's needed for joining. The end result would be a metric with the same values as cAdvisor's one but with the network name defined by multus' 
- I can change the operation to use multiplication. Either is fine I think as the daemon's metric will always be 0
- The number is `# of pods * num of interfaces` (which is still significant). I wasn't sure about the overhead vs the ease of use, as it took a bit to put the query together, but probably is straightforward. If that is the case, I can remove the recording rules.

Reply:
I think documenting the queries from those recording rules might be a good idea, but agree, if we don't use them anywhere we should not create them.

Reply:
The idea was to make this more usable / discoverable, but I agree with the overhead. Probably documenting the queries alongside multus documentation is enough and less resource-intensive. Will drop the recording rules.